dapps/marketplace/src/components/growth/ActionGroup.js

```diff
@@ -40,6 +40,7 @@ function ActionGroup(props) {
     }
   } else if (type === 'follows') {
     locked = allActionsInactive
+    locked = false // TODO DELETE!
```

Comment:
Not deleted?

Reply:
Wow you are fast thanks!
yup just noticed it myself and deleted it.